### PR TITLE
Update `srun` use case

### PR DESCRIPTION
**Pull request by Premas**
# Pull Request

**DO NOT WORK ON THIS PULL REQUEST**

We had to perform git surgery and this PR couldn't be salvaged. Use #600 instead.

## Overview
This section illustrates `srun` use case in running interactive and parallel jobs. The need and limitations of using `srun` is postulated. 

## Proposed Changes
1. Interactive job run testing using `srun`
2. `srun` used to run parallel jobs
3. Limitation of `srun`

## Related Issues
Related issue #557

### CODE REVIEW

File: docs/cheaha/slurm/submitting_jobs.md [hunk 1]
```diff
@@ -145,8 +145,68 @@ For more details on using `sbatch` please see the [official documentation](https
...
+```bash
+#!/bin/bash
+#SBATCH --nodes=2
+#SBATCH --ntasks-per-node=1
+#SBATCH --job-name=srun_test
+#SBATCH --partition=long
+#SBATCH --time=0:05:00
+#SBATCH --mem=4G
+
+srun hostname
```
Comment (mdefende) [inline docs/cheaha/slurm/submitting_jobs.md:195]: can we add an example where srun is partitioning the requested resources to multiple subtasks?

Something like:

srun --ntasks=1 --mem=2G task1
srun --ntasks=1 --mem=2G task2

Comment (Premas) [reply docs/cheaha/slurm/submitting_jobs.md:184]: Thanks for the suggestion, Matt. I have added an example explaining the partitioning of batch job resources to run simultaneous tasks using `srun`.

Comment (Premas) [reply docs/cheaha/slurm/submitting_jobs.md:184]: Please review the changes.

**Comment by wwarriner** [comment]
Old leftover from git surgery, not needed.

**Pull request closed without merging**